ISICbalance/ViewModels/BalanceViewModel.swift

```diff
@@ -13,32 +13,28 @@ import Alamofire
 import UIKit
 
 protocol BalanceViewModeling {
-    var balance: Property<String> { get }
-    
+    var balance: Property<Int> { get }
     var actions: BalanceViewModelingActions { get }
 }
 
 protocol BalanceViewModelingActions {
-    var getBalanceAction: Action<(),Balance,RequestError> { get }
+    var getBalance: Action<(),Balance,RequestError> { get }
 }
 
 extension BalanceViewModelingActions where Self: BalanceViewModeling {
     var actions: BalanceViewModelingActions { return self }
 }
 
-class BalanceViewModel: BaseViewModel, BalanceViewModeling, BalanceViewModelingActions {
+final class BalanceViewModel: BaseViewModel, BalanceViewModeling, BalanceViewModelingActions {
     typealias Dependencies = HasRequestManager
-    private let dependencies: Dependencies // TODO: není potřeba
 
-    lazy var balance = Property<String>(initial: "0 Kč", then: getBalanceAction.values.map { $0.balance }) // TODO: kč lokalizovat a přilepovat až ve view a vůbec.. možná spíš vracet číslo než string
-    let getBalanceAction: Action<(),Balance,RequestError>
+    lazy var balance = Property<Int>(initial: 0, then: getBalance.values.map { $0.balance }) // TODO: možná spíš vracet číslo než string -> Opravdu to mám převádět pomocí formatteru do double a pak to zase formátovat do double s čárkou?
```

Comment:
O tomhle jsme přece mluvili, že asi může zůstat i string, ale úplně správně by bylo to vyparsovat a pak lokalizovaně vypsat :-)

Reply:
Už jsem to převedl do intu a použil numberformatter na cenu a měnu 💪🏼